src/v3/src/bridge/legacy_nnvm/ir.cc

```diff
@@ -58,57 +65,58 @@ static void PrintIndexedGraph(const nnvm::Graph &g) {
     std::string op_name = op ? op->name : "None";
     if (input_nodes.count(i)) {
       input_cnt += 1;
-      op_name = (op ? op->name + " [input " : "[input ") + std::to_string(input_cnt) + "]";
+      op_name = (op ? op->name + " [input " : "[input ") +
+                std::to_string(input_cnt) + "]";
     } else {
       op_name = op ? op->name : "None";
     }
     std::cout << "  i = " << i << ", op = " << op_name
```

Comment:
LOG?

Reply:
This is piece of program that basically did visualization, and will be removed once the entire pipeline is done. So probably `cout` looks more casual and unlikely to forget :-)